web/pom.xml

```diff
@@ -131,11 +131,6 @@
 		 <groupId>org.apache.velocity</groupId>
 		 <artifactId>velocity-tools</artifactId>
 	  </dependency>
-	  <dependency>
```

Comment:
@rkorytkowski Didn't you asked to remove this dependency in one of your comments on outdated diff 8 days ago? Or did I mistunderstood which dependency should I remove?